test/tools/schema_validator/validator.cc

```diff
@@ -0,0 +1,62 @@
+#include "test/tools/schema_validator/validator.h"
+
+#include "common/router/config_impl.h"
+
+#include "test/test_common/printers.h"
+
+#include "tclap/CmdLine.h"
+
+namespace Envoy {
+
+const std::string Schema::ROUTE = "route";
+
+const std::string& Schema::toString(Type type) {
+  switch (type) {
+  case Type::Route:
+    return ROUTE;
```

Comment:
Is the plan to generalize this to other schemas? If it's just for offline route validation, I think this is already done in https://github.com/lyft/envoy/tree/master/test/tools/router_check.